common/changes/@itwin/core-backend/aks-query-reader-metadata_2024-03-15-12-03.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Added tests to verify access string information in QueryReader metadata.",
```

these are user facing, and they shouldnt care about tests

```suggestion
      "comment": "",
```